src/app/stack/utils/report-summary-utils.ts

```diff
@@ -86,7 +86,17 @@ export class ReportSummaryUtils {
                 temp: SecurityInformationModel = null;
 
             let analyzedDependencies: Array < ComponentInformationModel > ;
-            analyzedDependencies = userStackInfo.analyzed_dependencies;
+
+            // filter-out transitives which is also listed as direct
+            const uniqueDependencies: Set<string> = new Set<string>();
+            analyzedDependencies = userStackInfo.analyzed_dependencies.filter(info => {
```

Comment:
i think we can avoid this using `loadash` as it's already in application

`_.uniqBy(analyzed_dependencies, v => JSON.stringify([v.name, v.version]));`

WDYT?

Reply:
`loadash` is used as a devDependency, and I'm open to use it as a runtime dependency if you feel it would add value :)

Reply:
@invincibleJai , modified it, PTAL.